assembl/static2/js/app/components/debate/common/announcement.jsx

```diff
@@ -5,40 +5,55 @@ import { Col, Tooltip } from 'react-bootstrap';
 
 import StatisticsDoughnut from '../common/statisticsDoughnut';
 import { sentimentDefinitionsObject } from './sentimentDefinitions';
+import type { SentimentDefinition } from './sentimentDefinitions';
 import Media from '../../common/media';
 import { PublicationStates } from '../../../constants';
 import { multiColumnMapping } from '../../../utils/mapping';
 import PostsAndContributorsCount from '../../common/postsAndContributorsCount';
 
-export const createTooltip = (sentiment: Object, count: number) => {
+export const createTooltip = (sentiment: SentimentDefinition, count: number) => {
   return (
     <Tooltip id={`${sentiment.camelType}Tooltip`} className="no-arrow-tooltip">
       {count} <Translate value={`debate.${sentiment.camelType}`} />
     </Tooltip>
   );
 };
 
-export const getSentimentsCount = (posts: Object) => {
-  const counters = { ...sentimentDefinitionsObject };
+type SentimentsCounts = {
+  [string]: {
+    ...SentimentDefinition,
+    count: number
+  }
+};
+
+type Posts = {
+  edges: Array<{ node: { sentimentCounts: { [string]: number }, publicationState: string } }>
+};
+
+export const getSentimentsCount = (posts: Posts) => {
+  const counters: SentimentsCounts = { ...sentimentDefinitionsObject };
   Object.keys(counters).forEach((key) => {
     counters[key].count = 0;
   });
-  posts.edges.forEach(({ node: { sentimentCounts, publicationState } }) => {
-    if (publicationState === PublicationStates.PUBLISHED) {
-      Object.keys(counters).forEach((key) => {
-        counters[key].count += sentimentCounts[key];
-      });
-    }
-  });
-  return counters;
+  return posts.edges
+    .filter(({ node: { publicationState } }) => {
+      return publicationState === PublicationStates.PUBLISHED;
+    })
```

Comment:
#226 was merged in develop and develop merged into syntheses. Can you please update the condition?